src/test/java/uk/gov/hmcts/reform/userprofileapi/service/impl/UserProfileCreatorTest.java

```diff
@@ -335,4 +341,81 @@ public void test_consolidateRolesFromXuiAndIdam_3() {
         assertThat(rolesToUpdate.size()).isEqualTo(0);
     }
 
+    @Test
+    public void should_reinvite_user_successfully() {
+
+        Mockito.when(idamService.registerUser(any())).thenReturn(idamRegistrationInfo);
```

Comment:
As per QA Concerns please remove _Mockito._ and replace with static import

Reply:
done.